strings.go

```diff
@@ -18,11 +18,33 @@ func obfuscateLiterals(files []*ast.File) []*ast.File {
 			return true
 		}
 
-		// constants are not possibly if we want to obfuscate literals, therfore
-		// remove all constants and replace them by variables
-		if t.Tok == token.CONST {
-			t.Tok = token.VAR
+		if t.Tok != token.CONST {
+			return true
 		}
+
+		for _, spec := range t.Specs {
+			s, ok := spec.(*ast.ValueSpec)
+			if !ok {
+				// cannot happen because ast.GenDecl with token.Const are always of type ast.ValueSpec
+				return false
+			}
```

Comment:
since this can never happen, just do `spec := spec.(*ast.ValueSpec)` - the shadowing should work, so you don't need another name like `s`

Reply:
Hm, did you forget to apply this? I'll just do it in master.